features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/item-implement/template.jag

```diff
@@ -695,6 +695,32 @@
 
 													</div>
 												<% }; %>
+												    <% if(api.wsdl){
```

Comment:
slight formatting issue if condition went right side and we need spaces  in if statements  before {